include/mbedtls/x509_crt.h

```diff
@@ -348,6 +459,30 @@ int mbedtls_x509_crt_parse_file( mbedtls_x509_crt *chain, const char *path );
 int mbedtls_x509_crt_parse_path( mbedtls_x509_crt *chain, const char *path );
 #endif /* MBEDTLS_FS_IO */
 
+/**
+ * \brief          Parses a subject alternative name raw item
+ *                 to an identified structure;
+ *
+ * \param san_raw  The buffer hlding the raw data of the subject alternative name.
+ *
+ * \param san      The parsed subject alternative name.
+ *
+ * \note           Only "dnsName" and "otherName" of type hardware_module_name,
+ *                 as defined in RFC 4180 is supported.
+ *
+ * \note           This function should be called on a single raw data of
+ *                 subject alternative name. For example, after a call to
+ *                 mbedtls_x509_crt_get_subject_alt_names() one must iterate
+ *                 on every item in the sequence, and send it as parameter to
+ *                 this function.
+ *
+ * \return         Zero for success, \p MBEDTLS_ERR_X509_FEATURE_UNAVAILABLE
```

Comment:
Minor: Use multiple `\return` sections.
Minor: Use `\c 0 on success` instead of `Zero for success`
Minor: `#MBEDTLS_ERR_X509_FEATURE_UNAVAILABLE` instead of `\p MBEDTLS_...`.